src/components/generators/view-components/string.vue

```diff
@@ -33,6 +57,13 @@ export default {
   methods: {
     displayName (item) {
       return item.label || item.name || item.id
+    },
+    copyOnClipboard (link, event) {
+      navigator.clipboard.writeText(link)
+      this.copyPidShown = true
+      setTimeout(() => {
```

Comment:
it is better to assign the setTimeOut to a variable and call clearTimeout when the function inside has ended (memory leak avoidance), in this case after this.copyPidShown assignment

Reply:
Ok, I changed it